### PR TITLE
W 12652817 activity library cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
++
+Opens a window that enables you to create, edit, and delete tags. 
+* (image:activity-library-explorer-toolbar-import.png["The Import button.", 2%, 2%]) *Import*
++
+Opens a dialog that enables you to import an Activity Library Workflow package `(.calw)` file from your file system to the Activity Library.
+* (image:activity-library-explorer-toolbar-export.png["The Export button.", 2%, 2%]) *Export*
++
+Opens a dialog that enables you to export an Activity Library Workflow from the Activity Library to your file system as a `(.calw)` package file.
++
+When you export Activity Library workflows to a file, RPA Builder removes the passwords stored in wizards of Action Steps or User Account Variables to prevent passwords from being shared.
```
Comment (ms-jscholten) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:47]: **suggestion**: Not only passwords, but all sensitive information in Action Steps (this includes for example secrets in OAuthCredentials or AWS Credentials). If any Activity Parameters are used in the ALW, all sensitive information is also removed from the default values.

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
+
+A particular case is the Start event (Trigger). Although it is not an activity in the strict sense, a workflow is still assigned to this item, which initially starts the process. You can upload this event to the Activity Library.
```
Comment (ms-jscholten) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:5]: **issue**: Trigger Workflow does no longer exist. This was a mechanism in Legacy X1, but is no longer used in MuleSoft RPA

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+== Import an Activity from the Activity Library 
+
+To import an activity from the Activity Library to your project: 
+
+. Select an activity in your project. 
+. In the context pad, click *Import Workflow from Activity Library* (image:import-workflow-from-activity-library-icon.png["The Import Workflow from Activity Library Icon.", 2%, 2%])
+. In the window that opens, select the activity to import and click either *Add as a library reference* or *Add as a local copy*, depending on which method you want to use. 
++
+The Activity Library Explorer shows only the appropriate types of activities you can import, depending on which element you selected in the workflow.
```
Comment (ms-jscholten) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:42]: **issue**: This part is no longer relevant, because the trigger workflow does no longer exist. There is only one type of ALW now, that can be used across all available activities.

File: rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc [hunk 4]
```diff
@@ -205,7 +205,7 @@ _View_ -> _Show System Parameters_.
 
 == Explorer Pane
 
-In the window pane on the left-hand side, you will find five tabs
+In the window pane on the left-hand side there are find five tabs
```
Comment (ms-jscholten) [inline rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc:208]: **issue**: typo - "find" should be removed

**Review by ms-jscholten — CHANGES_REQUESTED** [review]

**Review by ms-jscholten — DISMISSED** [review]

File: rpa-builder/6.2/modules/ROOT/images/referenced-activity.png
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/images/referenced-activity.png:1]: Should this be "Log in to Salesforce"?
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22loginto%22

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
+= Activity Library Explorer 
+
+In the Activity Library Explorer you can view all the activity templates assigned to your organization. All users in your organization with the privilege *Activity Library Administration* have access to the same Activity Library.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:3]:
```suggestion
In the Activity Library Explorer, you can view all the activity templates assigned to your organization. All users in your organization with the privilege *Activity Library Administration* have access to the same Activity Library.
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+If another user adds an Activity, you see the change after refreshing the Activity Library Explorer.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:5]: Is the init cap necessary here? Maybe since it's not specifically referring to the UI, it can be "activity"?

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+image::activity-library-explorer.png["The Activity Librar Explorer window."]
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:7]:
```suggestion
image::activity-library-explorer.png["The Activity Library Explorer window."]
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+== Toolbar
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:9]: I realize that this is a "tour" of the UI, but we sort of want to move away from UI-centric docs and toward user-task docs. I'm wondering if it's necessary to document the toolbar?

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:9]: You could flip it around and have the user tasks first and point to the toolbar for how to accomplish each task.

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+If another user adds an Activity, you see the change after refreshing the Activity Library Explorer.
+
+image::activity-library-explorer.png["The Activity Librar Explorer window."]
+
+== Toolbar 
+
+The Activity Library Explorer toolbar contains the following elements: 
+
+* (image:activity-library-explorer-toolbar-refresh.png["The Refresh button.", 2%, 2%]) *Refresh*
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:13]: If this is obvious, maybe you don't need it?

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
++
+Updates the contents of the Activity Library Explorer.
+* (image:activity-library-explorer-toolbar-show-outline.png["The Show Outline button.", 2%, 2%]) *Show Outline*
++
+Shows an outline of the workflow.
+* (image:edit-icon.png["The Edit button.", 2%, 2%]) *Edit*
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:19]: obvious?

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:19]: I'll stop marking this, but something to think about.

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
++
+Opens the individual editing mode of an activity to make changes.
++
+If you upload the activity after making changes, the Activity Library saves the updated activity as a new revision.
++
+You can edit the Activities in the Activity Library regardless of which phase the Process that contains these activities is in.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:25]:
```suggestion
You can edit the activities in the Activity Library regardless of which phase the process that contains these activities is in.
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+* (image:activity-library-explorer-toolbar-show-log.png["The Show Log button.", 2%, 2%]) *Show Log*
++
+Opens a window containing all revisions of the activity. You can select a revision and edit it by choosing *Edit* in the individual editing mode.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:28]: Kim prefers select over choose in this case. https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:28]: Oh I see. You have select in the sentence already. Is *Edit* a button to click?

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+* (image:copy-icon.png["The Copy button.", 2%, 2%]) *Copy*
++
+Creates a copy of the selected element, which shows in the Activity Library.
++
+By default, this process copies all information such as name, description, and activity tags. During the upload, the Activity Library prefixes the activity name with _Copy of_. You can later edit this information as required.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:33]:
```suggestion
By default, this process copies all information, such as name, description, and activity tags. During the upload, the Activity Library prefixes the activity name with _Copy of_. You can edit this information later, if required. 
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
++
+Creates a copy of the selected element, which shows in the Activity Library.
++
+By default, this process copies all information such as name, description, and activity tags. During the upload, the Activity Library prefixes the activity name with _Copy of_. You can later edit this information as required. 
+* (image:delete-icon-alt.png["The Delete from Library button.", 2%, 2%]) *Delete from Library*
++
+Deletes the selected element from the Activity Library.
+* (image:activity-library-explorer-toolbar-manage-activity-tags.png["The Manage Activity Tags button.", 2%, 2%]) *Manage Activity Tags*
++
+Opens a window that enables you to create, edit, and delete tags.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:39]: Suggest removing the "Opens a window" in favor of just "Enables you to..."

Same for lines 42, 45.

((I think) The user cares about what they can do, not what the UI will do.)

File: rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
++
+Opens a window that enables you to create, edit, and delete tags. 
+* (image:activity-library-explorer-toolbar-import.png["The Import button.", 2%, 2%]) *Import*
++
+Opens a dialog that enables you to import an Activity Library Workflow package `(.calw)` file from your file system to the Activity Library.
+* (image:activity-library-explorer-toolbar-export.png["The Export button.", 2%, 2%]) *Export*
++
+Opens a dialog that enables you to export an Activity Library Workflow from the Activity Library to your file system as a `(.calw)` package file.
++
+When you export Activity Library workflows to a file, RPA Builder removes passwords and all sensitive information stored in wizards of Action Steps, User Account Variables, and Activity Parameters (for example secrets in OAuthCredentials or AWS Credentials) to prevent this information from being shared.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:47]:
```suggestion
When you export Activity Library workflows to a file, RPA Builder removes passwords and all sensitive information stored in wizards of Action Steps, User Account Variables, and Activity Parameters (for example, secrets in OAuthCredentials or AWS Credentials) to prevent this information from being shared.
```

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc:47]: I assume "wizards of..." makes sense to the user  :)

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 16]
```diff
@@ -0,0 +1,68 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:3]:
```suggestion
The Activity Library enables you to reuse activities that repeat during the automation of business processes, such as a login, to avoid creating the same workflow multiple times.
```

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:3]: (Not sure if that edit changes the meaning.)

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+Use the Activity Library Explorer to edit, copy, or delete activities and search for activities in your organization's library based on Activity Tags, which you can create and assign to the activities to further categorization and filtering capabilities.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:5]: Long sentence. Is there a way to split into two?

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
++
+For more information about assigning user privileges, see xref:rpa-manager::usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
+
+== Upload an Activity to the Activity Library
+
+To upload an activity to the Activity Library:
+
+. In RPA Builder, open a process for editing. 
+. Click an activity to select it and open the context pad. 
+. In the context pad, click *Add to Activity Library* (image:add-to-activity-library-icon.png["The Add to Activity Library Icon.", 2%, 2%])
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:20]: For all of these alt texts that aren't complete sentences - remove the terminating punctuation.

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
++
+image:context-pad.png["The context pad of an activity.", 50%, 50%]
+. In the *Upload Workflow to Activity Library* window that opens, provide a name and a description for the activity.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:23]:
```suggestion
. In the *Upload Workflow to Activity Library* window, provide a name and a description for the activity. 
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+
+To import an activity from the Activity Library to your project: 
+
+. Select an activity in your project. 
+. In the context pad, click *Import Workflow from Activity Library* (image:import-workflow-from-activity-library-icon.png["The Import Workflow from Activity Library Icon.", 2%, 2%])
+. In the window that opens, select the activity to import and click either *Add as a library reference* or *Add as a local copy*, depending on which method you want to use. 
+
+** *Add as a library reference*
++
+This method creates in each process a link to an activity from the Activity Library. Because the activity is already saved on the library, it is not uploaded again. The activity automatically receives the name of the reference, and you cannot change the name in the BPMN editor. An activity with a referenced workflow shows a gray background in the task element.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:42]:
```suggestion
This method creates a link in each process to an activity from the Activity Library. Because the activity is already saved in the library, it is not uploaded again. The activity automatically receives the name of the reference, and you cannot change the name in the BPMN editor. An activity with a referenced workflow shows a gray background in the task element. 
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
++
+You cannot import an activity that contains activity parameters that use the same name but a different data type as the activity parameters in the process.
+
+** *Add as a local copy*
++
+This method creates a local copy of the workflow behind the activity and inserts it into the project, which enables you to make changes. However, because there is no reference to the activity in the Activity Library, the local copy does not automatically update if the activity in the Activity Library changes. 
+
+== Convert Referenced Activities to a Local Copy
+
+Creating a copy of a referenced activity enables you to modify such activity.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:56]:
```suggestion
Creating a copy of a referenced activity enables you to modify the activity. 
```

Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:56]:
```suggestion
Create a copy of a referenced activity to modify it. 
```

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:56]: or maybe...

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 22]
```diff
@@ -0,0 +1,68 @@
...
+This method creates a local copy of the workflow behind the activity and inserts it into the project, which enables you to make changes. However, because there is no reference to the activity in the Activity Library, the local copy does not automatically update if the activity in the Activity Library changes. 
+
+== Convert Referenced Activities to a Local Copy
+
+Creating a copy of a referenced activity enables you to modify such activity. 
+
+To create a local copy of a referenced activity:
+
+. In the BMPN editor, double click the referenced activity you want to copy.
+. In the dialog that shows, click *Create Copy*.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:61]: In general, I think you can leave off "In the dialog...." and just say "Click..."

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 23]
```diff
@@ -0,0 +1,68 @@
...
+
+After you click *Create a Copy*, the workflow behind the activity opens in the Workbench for editing. Because the activity loses its reference, the next time you upload the activity it is treated as a new workflow.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:63]:
```suggestion
After you click *Create a Copy*, the workflow behind the activity opens in the Workbench for editing. Because the activity loses its reference, the next time you upload the activity, it is treated as a new workflow. 
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 24]
```diff
@@ -0,0 +1,68 @@
...
+To create a local copy of a referenced activity:
+
+. In the BMPN editor, double click the referenced activity you want to copy.
+. In the dialog that shows, click *Create Copy*.
+
+After you click *Create a Copy*, the workflow behind the activity opens in the Workbench for editing. Because the activity loses its reference, the next time you upload the activity it is treated as a new workflow. 
+
+== See Also 
+
+* xref:activity-library-explorer.adoc[Activity Library Explorer]
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:67]: I've been suggesting that folks not hard-code the titles in links unless they want the text to be different from the title. Just easier for maintenance when titles change...

File: rpa-builder/6.2/modules/ROOT/pages/activity-tags.adoc [hunk 25]
```diff
@@ -0,0 +1,34 @@
...
+. Provide a descriptive name to the tag and click *OK*.
++
+The name of the tag must be unique. 
+
+[WARNING]
+If you delete tags, they are removed automatically from all assignments.
+
+== Assign an Activity Tag
+
+After you have created tags, you can assign them to a workflow when you upload the workflow to
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-tags.adoc:22]:
```suggestion
After you created tag, you can assign them to a workflow when you upload the workflow to
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-tags.adoc [hunk 26]
```diff
@@ -0,0 +1,34 @@
...
+the Activity Library by using the *Activity Tags* tab.
+
+image::upload-workflow-to-activity-library-activity-tags.png["The Upload Workflow to Activity Library window showing the Activity Tags tab." 50%, 50%]
+
+Select an Activity tag and click (image:add-tag.png[The add tag button, 1%, 1%]) to add it to the workflow, or click (image:remove-tag.png[The remove tag button, 1%, 1%]) to remove the tag from the workflow.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/activity-tags.adoc:27]: do you need " " around the alt text?

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-tags.adoc:27]: Should these button names be init cap?

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc [hunk 27]
```diff
@@ -54,7 +54,9 @@ The paths calculated at runtime for File Operations are checked against the list
 
 * *Path / file name / invalid characters*
 +
-The path and file name are managed separately and are made available to other operations as the Outbound Variables *File Name* and *Directory Path*. None of the Action Steps are able to interpret wildcards in the form of `?` or `*`. Use unique file names at all times. If the path or file name contains invalid characters, the Action Step goes into an error state (this does not apply to the *File name* in the *Iterate over Files Action Step*). The Iterate over Files Action Step iterates over the result of a wildcard operation in the form of an Action Block.
+The path and file name are managed separately and are made available to other operations as the Outbound Variables *File Name* and *Directory Path*. None of the Action Steps are able to interpret wildcards in the form of `?` or `*`.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc:57]: any way to make this active voice? 
Should Outbound Variables be lowercase?

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc:57]: Action Steps cannot interpret wildcards in the form of `?` or `*`. 
or
Action Steps don't support the `?` or `*` wildcards.

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc [hunk 28]
```diff
@@ -54,7 +54,9 @@ The paths calculated at runtime for File Operations are checked against the list
 
 * *Path / file name / invalid characters*
 +
-The path and file name are managed separately and are made available to other operations as the Outbound Variables *File Name* and *Directory Path*. None of the Action Steps are able to interpret wildcards in the form of `?` or `*`. Use unique file names at all times. If the path or file name contains invalid characters, the Action Step goes into an error state (this does not apply to the *File name* in the *Iterate over Files Action Step*). The Iterate over Files Action Step iterates over the result of a wildcard operation in the form of an Action Block.
+The path and file name are managed separately and are made available to other operations as the Outbound Variables *File Name* and *Directory Path*. None of the Action Steps are able to interpret wildcards in the form of `?` or `*`. 
++
+Use unique file names at all times. If the path or file name contains invalid characters, the Action Step goes into an error state (this does not apply to the *File name* in the *Iterate over Files Action Step*). The Iterate over Files Action Step iterates over the result of a wildcard operation in the form of an Action Block.
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc:59]: Is it possible to lc Action Step? It's hard to parse this:  The Iterate over Files Action Step 

Is there a way to simplify this - it's unclear to me what it means:
The Iterate over Files Action Step iterates over the result of a wildcard operation in the form of an Action Block.

File: rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc [hunk 29]
```diff
@@ -205,7 +205,7 @@ _View_ -> _Show System Parameters_.
 
 == Explorer Pane
 
-In the window pane on the left-hand side, you will find five tabs
+In the window pane on the left-hand side there are five tabs
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc:208]: For A11Y, avoid directional language.

Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc:208]:
```suggestion
The navigation pane includes five tabs 
```

Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc:208]: (probably wrong)

File: rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc [hunk 30]
```diff
@@ -253,7 +253,7 @@ downloaded for editing.
 If you use the *_Activity Library,_* you can view and delete all available
 Activity Templates here.
 
-//For a more detailed description of how to use the Activity Library, see the chapter Activity Library.
+For a more detailed description of how to use the Activity Library, see xref:activity-library.adoc[Using the Activity Library]
```
Comment (hannanelson) [inline rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc:256]:
```suggestion
For a more detailed description of how to use the Activity Library, see xref:activity-library.adoc[Using the Activity Library].
```

**Review by hannanelson — APPROVED** [review]

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc (hunk 18 above)
Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:20]: And probably the "The"s

File: rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc (hunk 27 above)
Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/toolbox-file-operations.adoc]: I still think this is awkward: 
None of the Action Steps are able to interpret wildcards in the form of `?` or `*`.

File: rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc (hunk 29 above)
Comment (hannanelson) [reply rpa-builder/6.2/modules/ROOT/pages/ui-overview.adoc]: Please reword to avoid "left-hand"

**Review by dmerlob — APPROVED** [review]
